src/main/java/io/openliberty/tools/intellij/actions/LibertyDevStopAction.java

```diff
@@ -9,19 +9,29 @@
  *******************************************************************************/
 package io.openliberty.tools.intellij.actions;
 
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.vfs.VirtualFile;
+import io.openliberty.tools.intellij.LibertyModule;
 import io.openliberty.tools.intellij.util.LibertyActionUtil;
 import io.openliberty.tools.intellij.util.LocalizedResourceUtil;
 import org.jetbrains.plugins.terminal.ShellTerminalWidget;
 
 public class LibertyDevStopAction extends LibertyGeneralAction {
 
-    public LibertyDevStopAction() {
-        setActionCmd(LocalizedResourceUtil.getMessage("stop.liberty.dev"));
+    /**
+     * Returns the name of the action command being processed.
+     *
+     * @return The name of the action command being processed.
+     */
+    protected String getActionCommandName() {
```

Comment:
Seeing this method in a lot of Actions. Would it be good to refactor this method into the abstract class along with setting a String to store the message key?

Reply:
Hi Evie, the getActionCommandName() is an abstract method that is already in the abstract class LibertyGeneralAction. It is just that there are 12 different different commands/actions that implement it. Note that we cannot store data inside the abstract class. Data caching is what led to the issue this PR is attempting to fix, and it goes against documented restrictions (see doc link above).